cli/test-tls/ca.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could you move these to `cli/testdata/tls/`?

We have a convention for test data (e.g. `test/integration/testdata`)